Code/Legacy/CrySystem/ViewSystem/DebugCamera.cpp

```diff
@@ -58,10 +58,10 @@ DebugCamera::~DebugCamera()
 ///////////////////////////////////////////////////////////////////////////////
 void DebugCamera::OnEnable()
 {
-    m_position = gEnv->pSystem->GetViewCamera().GetPosition();
+    m_position = Vec3_Zero; //  gEnv->pSystem->GetViewCamera().GetPosition();
```

Comment:
Please remove comments here, though this is fully legacy and should be able to be removed later